Confluent.Kafka.sln

```diff
@@ -1,101 +1,101 @@
-		{BAB88872-01A1-41E7-98EC-95C240D076E8}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{BAB88872-01A1-41E7-98EC-95C240D076E8}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{BAB88872-01A1-41E7-98EC-95C240D076E8}.Release|Any CPU.Build.0 = Release|Any CPU
-		{D6053A17-99FA-4164-A8E1-8D24E196BED9}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{D6053A17-99FA-4164-A8E1-8D24E196BED9}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{D6053A17-99FA-4164-A8E1-8D24E196BED9}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{D6053A17-99FA-4164-A8E1-8D24E196BED9}.Release|Any CPU.Build.0 = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
-	GlobalSection(NestedProjects) = preSolution
-		{B2DDB635-4423-45D7-B3DC-F701E6010868} = {56321620-6DE2-44DC-942B-E3701C01EBFE}
-		{33151BE2-C10B-41BC-8C5E-E55211A1722D} = {03D93818-5A54-4913-A5CD-CA0C373CEE05}
-		{A7BF0A75-D3E7-4024-8597-5FCCC567D372} = {5E1F754C-FCE9-4B80-B13D-5F9ACE13305A}
-		{85F08EF0-9E30-49D3-B86F-DBA36267843C} = {5E1F754C-FCE9-4B80-B13D-5F9ACE13305A}
-		{2E6FC09E-2BED-4765-8298-9584E172B6CF} = {5E1F754C-FCE9-4B80-B13D-5F9ACE13305A}
-		{0A3DD19C-3A80-47DB-83B0-A1EF6078C9EE} = {5E1F754C-FCE9-4B80-B13D-5F9ACE13305A}
-		{9EA27B93-4714-453C-9319-1920945CEB78} = {5E1F754C-FCE9-4B80-B13D-5F9ACE13305A}
-		{DDCE9F6B-6A9D-4B05-BAD0-E77D6359B1FB} = {5E1F754C-FCE9-4B80-B13D-5F9ACE13305A}
-		{C3607ECF-7A8B-488C-B7D5-00388F1318A7} = {03D93818-5A54-4913-A5CD-CA0C373CEE05}
-		{BAB88872-01A1-41E7-98EC-95C240D076E8} = {03D93818-5A54-4913-A5CD-CA0C373CEE05}
-		{D6053A17-99FA-4164-A8E1-8D24E196BED9} = {03D93818-5A54-4913-A5CD-CA0C373CEE05}
-	EndGlobalSection
-EndGlobal
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
+VisualStudioVersion = 14.0.25420.1
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Confluent.Kafka", "src\Confluent.Kafka\Confluent.Kafka.xproj", "{B2DDB635-4423-45D7-B3DC-F701E6010868}"
+EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Confluent.Kafka.Test", "test\Confluent.Kafka.UnitTests\Confluent.Kafka.Test.xproj", "{33151BE2-C10B-41BC-8C5E-E55211A1722D}"
+EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "SimpleProducer", "examples\SimpleProducer\SimpleProducer.xproj", "{A7BF0A75-D3E7-4024-8597-5FCCC567D372}"
+EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "SimpleConsumer", "examples\SimpleConsumer\SimpleConsumer.xproj", "{85F08EF0-9E30-49D3-B86F-DBA36267843C}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{56321620-6DE2-44DC-942B-E3701C01EBFE}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "test", "{03D93818-5A54-4913-A5CD-CA0C373CEE05}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "examples", "examples", "{5E1F754C-FCE9-4B80-B13D-5F9ACE13305A}"
+EndProject

```

Comment:
ayay, whitespace diff.
I think you should set your git client to auto-convert Windows line-endings to Unix. see auto crlf.